testcases.py

```diff
@@ -17,7 +17,8 @@
 KB = 1 << 10
 MB = 1 << 20
 
-QUIC_VERSION = "0xff00001d"  # draft-29
+QUIC_DRAFT = 29  # draft-29
+QUIC_VERSION = hex(0xFF000000 + QUIC_DRAFT)
```

Comment:
What do we need this for?

Reply:
I was going to show which version the results are for on the web page using this.

Reply:
Makes sense. It's not implemented yet?

Reply:
No, I forgot :-) will add it now